doc/decisions/decision_process.md

```diff
@@ -33,13 +45,16 @@ Decisions need to:
 
 We base our decision process and template on:
 
-- [''using patterns to capture architectural decisions''](https://dl.acm.org/doi/10.1109/MS.2007.124)
-- [arc42 decisions](http://docs.arc42.org/section-9/), and
+- [''using patterns to capture architectural decisions''](https://dl.acm.org/doi/10.1109/MS.2007.124),
+- [arc42 decisions](http://docs.arc42.org/section-9/),
+- [ADR](https://adr.github.io/), and
 - [RFCs in rust-lang](https://github.com/rust-lang/rfcs).
 
 Following subsections describe all steps a decision might run through.
 Only two of them are mandatory.
 
+Explanations of the template are in [explanations.md](explanations.md).
```

Comment:
```suggestion
Explanations of the template are in [template_explanations.md](template_explanations.md).
```

Maybe we even call them `TEMPLATE.md` and `EXPLANATIONS.md` to show that they are not actual decisions. Also makes the files easier to spot in a list when you're looking for them.